.github/workflows/release-and-publish.yml

```diff
@@ -10,6 +10,11 @@ jobs:
   Release-Artifacts:
     name: Draft Release
     runs-on: ubuntu-latest
+    permissions:
+      actions: read
+      contents: write
+      deployments: read
+      packages: none
```

Comment:
do we need to specify `packages: none`? the docs indicate that 
> Once a permissions block is added, all other permissions are set to none by default except the metadata permissions which is read by default.

Reply:
Indeed, explicit was the intent for anyone in future if decides to add any GH level packages, the code level block should make this more clear. As an archive the default knowledge could be linked here: https://docs.opensource.microsoft.com/github/apps/permission-changes/

Basically a choice, so that its kind of explicit for some flag which don't need anyone new too much googling to understand the permission attributes.